packages/fxa-auth-server/bin/key_server.js

```diff
@@ -11,9 +11,17 @@ require('../lib/newrelic')();
 
 const jwtool = require('fxa-jwtool');
 const P = require('../lib/promise');
+const StatsD = require('hot-shots');
 
 function run(config) {
   const log = require('../lib/log')(config.log);
+  const statsd = new StatsD({
+    ...config.statsd,
+    errorHandler: err => {
+      // eslint-disable-next-line no-use-before-define
+      log.error('statsd.error', err);
+    },
+  });
```

Comment:
@lmorchard Added this to trap and log errors from statsd.

In a future patch the order the `log` and `statsd` above will be swapped so I can pass `statsd` into the `log` constructor.  That's the reason for the eslint comment I accidentally left in there. 😬